spicepy/_http.py

```diff
@@ -30,7 +30,8 @@ def __init__(self, base_url: str, headers: Dict[str, str]) -> None:
         self.session = self._create_session(headers)
 
         # set the x-spice-user-agent header
```

Comment:
```suggestion
        # set the user-agent header
```

Reply:
It seems we always use `SPICE_USER_AGENT` in this case - when it is possible?  Can there be a case that there will always be default user-agent even if custom was provided?